activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Add support for `Array#intersect?` to `ActiveRecord::Relation`.
```

Comment:
This file uses `intersect?` (without an 'S'), but the other files use `intersects?` (with an 'S').  Perhaps that can be corrected, to help avoid bugs when people copy the method name from the change log?

Reply:
Good catch. Fixed in https://github.com/rails/rails/commit/e446ef82c8174a4bd6c98f670912bb395b3091d6, thank you.

Reply:
The CHANGELOG is correct, I fixed the code in https://github.com/rails/rails/commit/119b4b2af1d848362a1a9c1cd8a2bea46700b154